Maquetado/scss/titles.scss

```diff
@@ -1,15 +1,20 @@
-header {
+.title {
   display: flex;
-  justify-content: center;
+  justify-content: flex-start;
+  margin: 48px 0 16px 0;
   margin-bottom: 16px;
 
-  h3{
+  &:first-child{
+    margin-top: 0;
+  }
+
+  h3 {
     align-self: center;
-    font-size: 32px;
-    margin-left: -24px;
+    font-size: 28px;
+    margin-left: -17.6px;
```

Comment:
mmm negative marges are not the best set the img as a ::before component and add the style to that element